modules/ROOT/pages/upgrade-message-flows.adoc

```diff
@@ -1,6 +1,6 @@
 = Upgrading Message Flow Runtime Templates
 
-Anypoint Partner Manager uses prebuilt templates to convert your B2B message flow configurations into runtime applications. As these templates are updated, you can upgrade your message flows to use them to take advantage of new capabilities that enable better integration with your trading partner's ecosystem.
+Anypoint Partner Manager uses prebuilt templates to convert your B2B message flow configurations into runtime applications. As these templates are updated, you can upgrade your message flows to use them to take advantage of new capabilities that enable better integration with your partner's ecosystem.
```

Comment:
This is similar (the same?) as another paragraph. Opportunity for reuse?

Is there any way to simplify the second sentence?